paymaster-analysis/utils/multisend.ts

```diff
@@ -0,0 +1,44 @@
+import { encodePacked, encodeFunctionData } from "viem";
+
+export type InternalTx = {
+  to: `0x${string}`;
+  data: `0x${string}`;
```

Comment:
wow... you learn something new every day! [Template type literals](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) are really cool!